### PR TITLE
Allow concurrent runs of unit_test.sh

### PR DESCRIPTION
**Pull request by Victoremepunto**
The script currently only uses the IMAGE_TAG to create names for the containers used in the `unit_test.sh` script, causing concurrent running instance using the same image to affect each other.

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Comment by LightOfHeaven1994** [comment]
Amazing work! Thanks @Victoremepunto !

File: scripts/unit_test.sh [hunk 1]
```diff
@@ -22,37 +28,38 @@ if [[ -n "$ghprbPullId" ]]; then
...
   -e POSTGRESQL_USER="$DATABASE_USER" \
   -e POSTGRESQL_PASSWORD="$DATABASE_PASSWORD" \
   -e POSTGRESQL_DATABASE="$DATABASE_NAME" \
-  "$POSTGRES_IMAGE" || echo "0")
+  "$POSTGRES_IMAGE"); then
 
-if [[ "$DB_CONTAINER_ID" == "0" ]]; then
   echo "Failed to start DB container"
   exit 1
+
```
Comment (romanblanco) [inline scripts/unit_test.sh:53]: extra line

Comment (Victoremepunto) [reply scripts/unit_test.sh]: thanks, removed!

**Review by romanblanco — APPROVED** [review]
LGTM :+1:

**Pull request merged**